olive/systems/azureml/aml_system.py

```diff
@@ -193,9 +193,6 @@ def _create_args_from_resource_path(self, rp: OLIVE_RESOURCE_ANNOTATIONS):
             return None
         asset_type = get_asset_type_from_resource_path(model_resource_path)
 
```

Comment:
Any reason to remove these lines? Then how to tell the assert type for aml job input?